src/terra-dev-site/doc/application/how-to.2/use-translated-strings.doc.mdx

```diff
@@ -1,25 +1,26 @@
 # How To Use Translated Strings
 
-Translated strings can be retrieved using the `intl` object provided by `injectIntl` HOC-generator provided by [react-intl](https://github.com/formatjs/react-intl/blob/master/docs/API.md#injectintl).
+Translated strings can be retrieved using the [ApplicationIntlContext](/components/terra-application/application/reference/contexts/application-intl-context). An `ApplicationIntlContext.Provider` is configured and provided by [ApplicationBase](/components/terra-application/application/reference/components/application-base). The strings will be translated based upon the `locale` prop provided to ApplicationBase.
+
+In addition to the ApplicationIntlContext, the components provided by react-intl can be used directly. However, the ApplicationIntlContext can be used by Hooks-based components to access i18n capabilities without having to use the `injectIntl` HOC-generator from react-intl.
 
 ```jsx
-import React from 'react';
-import { injectIntl, intlShape } from 'react-intl';
+import React, { useContext } from 'react';
+import { ApplicationIntlContext } from 'terra-application/lib/application-intl';
+import { FormattedMessage } from 'react-intl';
+
+const ExampleComponent = () => {
+  const applicationIntl = useContext(ApplicationIntlContext);
```

Comment:
I'm looking at the react-intl v3 upgrade guide. Their example for utilizing intl with hooks is actually a custom hook called `useIntl`. What if we followed this strategy to make the v3 upgrade feel seem-less when we swap ours out for theirs?

https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#new-useintl-hook-as-an-alternative-of-injectintl-hoc

Reply:
We could expose an additional `useIntl` hook, but we'd still need to create our own Context to do so.

The kicker here is that I wasn't planning on creating custom hooks for the various other contexts being exposed. Do you guys find them (in general) useful? It might be awkward to offer only one. They might be easier for  consumers to test with?

Reply:
IMO I think a custom hook is a nicity that doesn't really give extra to consumers. I don't think we need to provided a custom hook for our context APIs. My only thought here would be that the upgrade path would be simple

Reply:
I'd prefer to hold off on providing it for the time being and add it later if we need to based on consumer sentiment. Sound good to you?